deepgram/transcription.py

```diff
@@ -40,11 +66,30 @@ async def __call__(
 
 
 class LiveTranscription:
+    """
+    This class allows you to perform live transcription by connecting to Deepgram's Transcribe Streaming API.
+    It takes in options for the transcription job, and a callback function to handle events.
+
+    """
+
     _root = "/listen"
     MESSAGE_TIMEOUT = 1.0
 
     def __init__(self, options: Options,
                  transcription_options: LiveOptions) -> None:
+        """
+        The __init__ function is called when an instance of the class is created.
+        It initializes all of the attributes that are part of the object, and can be
+        accessed using "self." notation. In this case, it sets up a list to store any
+        messages received from Transcribe Streaming.
+        
+        :param self: Used to Reference the object instance of the class.
```

Comment:
Please remove this line from docstring `:param self: Used to Reference the object instance of the class.`